src/utils.ts

```diff
@@ -10,3 +10,15 @@ export function Override(pattern: string | RegExp, ...rest: any[]): void {
     // @ts-ignore
     defineStep(pattern, ...rest);
 }
+
+let world: IWorld;
+export function bindWorld(worldObject: IWorld) {
+    world = worldObject;
+}
+export async function executeStep(this: any, text: string) {
+    const stepDefsLibrary = supportCodeLibraryBuilder.buildStepDefinitions();
+    const step = stepDefsLibrary.stepDefinitions.find(s => s.matchesStepName(text));
```

Comment:
What if there are several step definitions match with text, wouldn't you ike to throw an error?